webapp/components/blockable_link/blockable_link.jsx

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) 2016-present Mattermost, Inc. All Rights Reserved.
```

Comment:
Also, could you maybe move `webapp/components/blockable_link` to `webapp/components/admin_console/blockable_link`? Since it's view state is specific to the admin console, I think it would be a good idea to keep the component in the same folder